conf/config.toml

```diff
@@ -51,6 +51,7 @@ max-store-down-time = "1h"
 leader-schedule-limit = 64
 region-schedule-limit = 16
 replica-schedule-limit = 24
+balance-tolerant-ratio = 2.5
```

Comment:
what does tolerant mean here?

Reply:
tolerate size difference

Reply:
so the better name may be tolerate-size-ratio